package.json

```diff
@@ -43,6 +44,12 @@
       }
     }
   },
+  "//": "This file is partially managed by code-skeleton. Changes may be overwritten.",
+  "tap": {
+    "coverage": true,
+    "ts": true
+  },
+  "types": "lib/index.d.ts",
```

Comment:
these lines are all due to the bug in code-skeleton where some top level fields were being ignored. that's been fixed and so these are getting added now